### ISSUE TITLE
[YSQL] DBeaver: ERROR:  Invalid argument: Index column 1 is already bound to another value

### ISSUE THREAD

**Issue by dacrypt** [yugabyte/yugabyte-db#3731]
Using Version 2.1 

Hi, I’m trying v2.1 locally. I’m getting this error in DBeaver when trying to see any table
```
SQL Error [XX000]: ERROR: Invalid argument: Index column 1 is already bound to another value
  ERROR: Invalid argument: Index column 1 is already bound to another value
  ERROR: Invalid argument: Index column 1 is already bound to another value
```
it was working fine with previous YB version

yugabyte-data/node-1/disk-1/yb-data/tserver/logs/postgresql-2020-02-25_121247.log
```
2020-02-25 12:42:33.369 -03 [12550] ERROR:  Invalid argument: Index column 1 is already bound to another value
2020-02-25 12:42:33.369 -03 [12550] STATEMENT:  SELECT c.relname,a.*,pg_catalog.pg_get_expr(ad.adbin, ad.adrelid, true) as def_value,dsc.description
	FROM pg_catalog.pg_attribute a
	INNER JOIN pg_catalog.pg_class c ON (a.attrelid=c.oid)
	LEFT OUTER JOIN pg_catalog.pg_attrdef ad ON (a.attrelid=ad.adrelid AND a.attnum = ad.adnum)
	LEFT OUTER JOIN pg_catalog.pg_description dsc ON (c.oid=dsc.objoid AND a.attnum = dsc.objsubid)
	WHERE NOT a.attisdropped AND c.oid=$1 ORDER BY a.attnum
```

**Comment by ndeodhar** [comment]
Possibly introduced by https://github.com/yugabyte/yugabyte-db/commit/82bb11c8456b4df620c5d190d8b5b6dd24482ac3.

**Comment by nocaway** [comment]
The failure is reproducible with the following statement.
```
  SELECT pg_catalog.pg_get_expr(ad.adbin, ad.adrelid, true)
    FROM pg_catalog.pg_attribute a
    INNER JOIN pg_catalog.pg_class c ON (a.attrelid = c.oid)
    LEFT OUTER JOIN pg_catalog.pg_attrdef ad ON (a.attrelid=ad.adrelid AND a.attnum = ad.adnum)
    WHERE c.oid=112;
```

**Comment by nocaway** [comment]
For the above SELECT, Postgres creates two entries for the key `c.oid` in the ScanKey.  YugaByte should LOG() this information instead of disallowing and rejecting duplicate key.

This has been fixed in master branch.  We'll patch it to 2.1 after running tests.

In the long run, we'll need to fix Postgres's original code not to have duplicate key in ScanKey

**Comment by ndeodhar** [comment]
Fix is now available in 2.1.
